core/server/ghost-server.js

```diff
@@ -18,7 +16,6 @@ function GhostServer(rootApp) {
     this.httpServer = null;
     this.connections = {};
     this.connectionId = 0;
-    this.upgradeWarning = setTimeout(this.logUpgradeWarning.bind(this), 5000);
```

Comment:
If you're going to remove this, the `clearTimeout` will also need to be removed.